src/Reducers/filter.js

```diff
@@ -0,0 +1,14 @@
+import { CHANGE_FILTER } from '../Actions';
+
+const filterReducer = (state = [], action) => {
+  let statenew = { ...state };
```

Comment:
- [ ] The state should default to `All` as per the requirements. [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/ItDefaultsToAll.png). Kindly update it to `const filterReducer = (state = 'All', action)`